modules/javafx.graphics/src/main/java/javafx/stage/StageStyle.java

```diff
@@ -32,12 +32,16 @@
 public enum StageStyle {
 
     /**
-     * Defines a normal {@code Stage} style with a solid white background and platform decorations.
+     * Defines a normal {@code Stage} style with a solid background and platform decorations.
```

Comment:
Should we say the background color is platform-specific, or perhaps identify why it is different?

Reply:
I restored it, I think it's a Linux glass specific bug.

Reply:
If it's a bug, could you please create a JBS ticket?

Reply:
Sure. I'll look into it.

Reply:
To be honest, I don't understand why we specify this at all.  "white" surely won't be the default when running on dark theme... I'm pretty sure on Windows it isn't even white, it's some light gray.

Reply:
When there's no scene attached, it seems to fall back to the default behavior defined by the OS. However, once a scene is set, a white background is sent, although this gets overridden by the root node of the scene.

So I’d say this behavior isn’t really tied to Stage state or StageStyle, but rather to the presence and characteristics of the Scene itself.